querydsl-apt/src/main/java/com/querydsl/apt/ExtendedTypeFactory.java

```diff
@@ -401,7 +401,11 @@ private Type createClassType(DeclaredType declaredType, TypeElement typeElement,
         return type;
     }
 
-    private Type createMapType(Iterator<? extends TypeMirror> typeMirrors, boolean deep) {
+    public boolean isSimpleTypeEntity(TypeElement typeElement, Class<? extends Annotation> entityAnn) {
```

Comment:
where does this method name come from? (might be ok i just dont see the origin/context)

Reply:
line 357, Spring Data JDBC only has one required annotation and that is @Id which is placed on id column and not on the entity class itself hence this change is needed.